src/main/java/core/basesyntax/handler/SupplyOperationHandler.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.handler;
+
+import core.basesyntax.db.Storage;
+
+public class SupplyOperationHandler implements ShopOperationHandler {
+    @Override
+    public void doOperation(String fruitName, String quantityToOperate) {
+        Integer oldQuantityValue = Storage.FRUIT_STORAGE.get(fruitName.toLowerCase());
+        Storage.FRUIT_STORAGE.put(fruitName.toLowerCase(),
+                Integer.parseInt(quantityToOperate) + oldQuantityValue);
```

Comment:
Here we should assume that supply may come before balance so this will be new position in our storage.
For this we should make null -> 0 conversion if met:
```suggestion
        Integer oldQuantityValue = Storage.FRUIT_STORAGE.get(fruitName.toLowerCase());
        oldQuantityValue = oldQuantityValue == null ? 0 : oldQuantityValue;
        Storage.FRUIT_STORAGE.put(fruitName.toLowerCase(),
                Integer.parseInt(quantityToOperate) + oldQuantityValue);
```

Reply:
Done